__tests__/src/services/ethereum/index.js

```diff
@@ -36,4 +41,18 @@ describe('ethereum', () => {
     const numCitizens = await nationsObject.getNumCitizens(0);
     expect(numCitizens.toNumber()).toEqual(0);
   });
+  test('Create contract test', async () => {
+    const wallet = new ethers.Wallet('0xefc27ba5330258fcfb75e28e4e6efd88458751086998bbfad99257035fb3e160');
+    wallet.provider = new providers.InfuraProvider('rinkeby');
+    ethereum = new Ethereum(wallet, 'dev');
+    // const txReceipt = await ethereum.deployContract(
```

Comment:
I guess we should remove this since it's commented code, no?

Reply:
Fixed